components/ScreenerView/ScreenerViewGene.vue

```diff
@@ -116,6 +116,7 @@
   import datasets from '~/refex-sample/datasets.json';
   import geneFilters from '~/refex-sample/gene_filters.json';
   import ScreenerViewGeneFilter from './ScreenerViewGeneFilter.vue';
+import filters from '~/static/filters.json';
```

Comment:
@maruk0chan 
'filters' is declared but its value is never read
do you need it here? please check it.